manifests/vm-import-operator/v0.0.1/v2v.kubevirt.io_vmimportconfigs_cr.yaml

```diff
@@ -0,0 +1,7 @@
+---
+apiVersion: v2v.kubevirt.io/v1alpha1
+kind: VMImportConfig
+metadata:
+  name: vm-import-operator-config
```

Comment:
since this is namespaced scope resource, does it make sense to add `namespace: kubevirt-hyperconverged` ?
or does the namespace should be decided by the context in which this resource will be installed ?
Should we add it as a template and generate its output to replace the namespace value ?

Reply:
I checked SSP and CDI CRs and there is no namespace.

Reply:
so i think we need to make sure it is being added to each release by adding it as 
`v2v.kubevirt.io_vmimportconfigs_cr.yaml.in` under `templates/vm-import-operator/VERSION`

Reply:
ok, will add